sample_configuration/disco_aws.ini

```diff
@@ -32,9 +32,9 @@ default_product_line=teamname
 default_ami_available_wait_time=600
 
 [test]
-hostclass=mhcdiscointegrationtests
-command=/opt/wgen/disco_integration_tests/bin/run_tests.sh
-user=disco_tester
+test_hostclass=mhcdiscointegrationtests
```

Comment:
Why do we need to update the default `test` section? Doesnt `hostclass` key work, if not wouldnt this be backward incompatible?